sources/packages/web/src/views/aest/SearchStudents.vue

```diff
@@ -2,32 +2,48 @@
   <full-page-container>
     <h2 class="color-blue">Search Students</h2>
     <v-row class="mt-5"
-      ><v-col><label for="appNumber">Application Number</label></v-col
-      ><v-col><label for="firstName">Given Names</label></v-col
-      ><v-col><label for="lastName">Last Name</label></v-col> <v-col></v-col
+      ><v-col>Application Number</v-col><v-col>SIN</v-col
+      ><v-col>Given Names</v-col><v-col>Last Name</v-col> <v-col></v-col
     ></v-row>
     <v-row
-      ><v-col
-        ><InputText
-          type="text"
+      ><v-col>
+        <v-text-field
+          density="compact"
           data-cy="appNumber"
+          variant="outlined"
           v-model="appNumber"
-          @keyup.enter="searchStudents()" /></v-col
-      ><v-col
-        ><InputText
-          type="text"
+          @keyup.enter="searchStudents"
+        />
+      </v-col>
+      <v-col>
+        <v-text-field
+          density="compact"
+          data-cy="sin"
+          variant="outlined"
+          v-model="sin"
+          @keydown.space.prevent
```

Comment:
I believe that we can allow spaces and remove them on the API side using the method `removeWhiteSpaces`.

Reply:
I will update